src/test/kotlin/io/csbroker/apiserver/e2e/ProblemApiControllerTest.kt

```diff
@@ -701,4 +707,66 @@ class ProblemApiControllerTest {
                 )
             )
     }
+
+    @Test
+    @Order(7)
+    fun `Assessment grading result`() {
+        // given
+        val urlString = "$PROBLEM_ENDPOINT/grade/{grading_history_id}/assessment"
+
+        val now = Date()
+        val email = "test2@test.com"
+
+        val accessToken = tokenProvider.createAuthToken(
+            email = email,
+            expiry = Date(now.time + 600000),
+            role = Role.ROLE_USER.code
+        )
+
+        val assessmentRequestDto = AssessmentRequestDto(
+            AssessmentType.BAD,
+            "키워드 채점 기준이 정확하게 적용되지 않은 것 같아요."
+        )
+
+        val assessmentRequestDtoString = objectMapper.writeValueAsString(assessmentRequestDto)
+
+        // when
+        val result = mockMvc.perform(
+            RestDocumentationRequestBuilders
+                .post(urlString, gradingHistoryId!!)
+                .contentType(MediaType.APPLICATION_JSON)
+                .content(assessmentRequestDtoString)
+                .header(HttpHeaders.AUTHORIZATION, "Bearer ${accessToken.token}")
+                .accept(MediaType.APPLICATION_JSON)
+        )
+
+        // then
+        result.andExpect(MockMvcResultMatchers.status().isOk)
+            .andExpect(MockMvcResultMatchers.content().string(CoreMatchers.containsString("success")))
+            .andDo(
+                document(
+                    "problems/grade/assessment",
+                    preprocessRequest(prettyPrint()),
+                    preprocessResponse(prettyPrint()),
+                    requestFields(
+                        fieldWithPath("assessmentType").type(JsonFieldType.STRING)
+                            .description("평가 의견 타입 ( 좋음 : GOOD, 나쁨 : BAD, 적당 : SOSO )"),
```

Comment:
```suggestion
                            .description("평가 의견 타입 ( 좋음 : GOOD, 나쁨 : BAD, 보통 : NORMAL )"),
```
만약 soso를 변경하신다면 여기도요!!